src/main/python/e2e_experimental_settings.py

```diff
@@ -31,34 +33,33 @@ def __init__(self, qa_data_xml, qrel, word_embeddings_file, index_path, model_fi
         self.w2v_cache = word_embeddings_file
         self.index_path = index_path
         self.model_file = model_file
-        self.cmd_root = "python src/main/python/run_trecqa.py -input {} -output e2e.run -qrel {} -index {} -w2v-cache {} -qa-model-file {} -model sm".\
+        self.cmd_root = "python src/main/python/run_trecqa.py -input {} -output e2e.run -qrel {} -index {} -w2v-cache {} -qa-model-file {} -model idf". \
             format(self.qa_data_xml, self.qrel, self.index_path, self.w2v_cache, self.model_file)
         self.eval_cmd_root = "./eval/trec_eval.9.0/trec_eval -m map -m recip_rank -m bpref {}".format(self.qrel)
-        self.rbp_cmd_root = "rbp_eval {}".format(self.qrel)
+        self.rbp_cmd_root = "rbp_eval/rbp_eval {}".format(self.qrel)
 
     def add_setting(self, setting):
         self.settings[setting.label] = setting
         self._setup_combinations()
 
-
     def _setup_combinations(self):
         all_settings = []
         for setting in self.settings.values():
             all_settings.append(setting.get_options())
 
         self.combinations = []
```

Comment:
e2e_experimental_settings.py belongs to @gauravbaruah but I'll adopt these changes in the updated PR.